ngx_http_pipelog_module.c

```diff
@@ -2218,6 +2211,14 @@ ngx_http_pipelog_logger_process_main (ngx_cycle_t *cycle) {
     sigaddset(&set, SIGCHLD);
     sigprocmask(SIG_BLOCK, &set, NULL);
 
+    ccf = (ngx_core_conf_t *)ngx_get_conf(cycle->conf_ctx, ngx_core_module);
+    if (geteudi() == 0) {
```

Comment:
I think you mean geteuid() and not geteudi() ?

Reply:
oops... typo, sorry.